src/server/replica.cc

```diff
@@ -25,12 +26,15 @@ extern "C" {
 #include "server/rdb_load.h"
 #include "util/proactor_base.h"
 
+ABSL_FLAG(bool, enable_multi_shard_sync, true, "Enable sync flows of multi shard commands");
+
```

Comment:
Its a bit misleading and sounds as if you support something additional instead of changing the behaviour

I'd describe it as "Execute multi shards commands on replica syncrhonized" or smth else